Automatic/README.md

```diff
@@ -29,11 +29,18 @@ This can make application a bit more snappier since big image data won't need to
 
 Following command will play an audio file whenever something is copied clipboard.
 
-### [Show Copy Time](show-copy-time.ini)
+### [Store Copy Time](store-copy-time.ini)
 
-Shows copy time of new items in tag.
+Store copy time of new items in a tag.
+
+To show the copied time the "Tags" plugin is required. Also you have to configure a tag such as the following:
+
+`Tag Name`: `\1`
+
+`Match`: `(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2})`
+
+`Style Sheet`: `font-size: 9pt; border: 1px solid #000; background: #000; padding: 0 2px`
```

Comment:
Looks useful but the tag configuration is **optional**. I personally use it only to highlight "important" tag and for the rest (date/time and source window title) use the default style (i.e. not configured).

Reply:
Oh I didn't know anymore that tags are visible by default...